smithy-trait-codegen/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+description = "Plugin for Generating Trait Code from Smithy Models"
+
+ext {
+    set("displayName", "Smithy :: Trait Code Generation")
+    set("moduleName", "software.amazon.smithy.traitcodegen")
+}
+
+dependencies {
+    implementation(project(":smithy-codegen-core"))
+}
+
+// Set up Integration testing source sets
+sourceSets {
+    create("it") {
+        compileClasspath += sourceSets.main.get().output +
+                configurations["testRuntimeClasspath"] +
+                configurations["testCompileClasspath"]
+
+        runtimeClasspath += output +
+                compileClasspath +
+                sourceSets.test.get().runtimeClasspath +
+                sourceSets.test.get().output
+
+        java {
+            srcDir("${layout.buildDirectory.get()}/integ/")
+        }
+
+        resources {
+            srcDirs(layout.buildDirectory.get().dir("generated-resources"))
```

Comment:
nit: get can be after `dir()` call.

Reply:
What is the difference? Filtering first and then getting?